plugin/src/main/java/com/iridium/iridiumcore/utils/Placeholder.java

```diff
@@ -11,8 +13,10 @@
 public class Placeholder {
 
     private final String key;
```

Comment:
Semi breaking change, nothing should reference the key directly, but key is without percent signs, formattedKey is with signs